Solution/Flight/Airport.cs

```
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Flight
+{
+    public class Airport
+    {
+        private protected List<Person> passengers = new List<Person>();
+        public void AddToBase(Person newPerson)
+        {
+            passengers.Add(new Person()
+            {
+                FirstName = newPerson.FirstName,
+                LastName = newPerson.LastName,
+                Patronymic = newPerson.Patronymic,
+                PassportData = newPerson.PassportData,
+                FlightNumber = newPerson.FlightNumber
+            });
+        }
+        public void AddToBaseAfterCheckIn(Person newPerson)
+        {
+            passengers.Add(new Person()
+            {
+                RegistrationStatus = newPerson.RegistrationStatus,
+                BoardingPass = newPerson.BoardingPass,
+                BaggageStatus = newPerson.BaggageStatus,
+                BaggageId = newPerson.BaggageId,
+                Terminal = newPerson.Terminal,
+                GateNumber = newPerson.GateNumber,
+            });
+        }
+        public Person CheckPassport(string passportDetails)
+        {
+            try
+            {
+                passportDetails = passportDetails.ToUpper();
+            }
+            catch { }
```

empty catch - is not a good way